xcode/Subconscious/Shared/Components/Notebook/Notebook.swift

```diff
@@ -908,15 +915,33 @@ struct NotebookModel: ModelProtocol {
                 environment: environment
             )
         }
+        
         if AppDefaults.standard.isNoosphereEnabled {
+            // Intercept profile visits and use the correct view
+            if slashlink.slug.isProfile() {
```

Comment:
Nit: perhaps this should be a guard clause? (Because it's an early exit.)

Non-blocking.